sormas-stats-sitrep/utils/caseExport.R

```diff
@@ -0,0 +1,190 @@
+    person_id,
+    hospitalization_id 
+  
+    FROM public.cases 
+    
+    WHERE deleted = FALSE 
+    and caseclassification != 'NO_CASE'
+    and reportdate between '", fromDate, "' and '", toDate, "' 
+    and disease = 'CORONAVIRUS' 
+                       ")
+  cases <- dbGetQuery(sormas_db,queryCases)
+  
+  
+  # Person table SQL query
+  queryPersons <- paste0("SELECT DISTINCT id AS person_id,
+    approximateAge AS age_person,
+    sex AS sex_person,
+    deathDate AS death_date_person,
+    causeOfDeathDisease AS cause_of_death_disease_person
+
+    FROM public.person
+
+    WHERE id IN (SELECT person_id FROM public.cases
+      WHERE deleted = FALSE 
+      and caseclassification != 'NO_CASE'
+      and reportdate between '", fromDate, "' and '", toDate, "' 
+      and disease = 'CORONAVIRUS')
+                         ")
+  # Only persons with a person_id that also appears in a case 
+  # are queried from the database
+  persons <- dbGetQuery(sormas_db,queryPersons)
+  
+  # Symptoms table SQL query
+  querySymptoms <- paste0("SELECT DISTINCT id AS symptoms_id,
+    onsetDate AS onset_date
+    
+    FROM public.symptoms
+    
+    WHERE id IN (SELECT symptoms_id FROM public.cases
+      WHERE deleted = FALSE 
+      and caseclassification != 'NO_CASE'
+      and reportdate between '", fromDate, "' and '", toDate, "' 
+      and disease = 'CORONAVIRUS')
+                          ")
+  # Only symptoms with a symptoms_id that also appears in a case 
+  # are queried from the database
+  symptoms <- dbGetQuery(sormas_db, querySymptoms)
+  
+  # Hospitalizations table query 
+  queryHospitalizations <- paste0("SELECT DISTINCT id AS hospitalization_id,
+    admittedToHealthFacility AS admitted_to_health_facility,
+    admissionDate AS admission_date,
+    hospitalizationReason AS hospitalization_reason
+    
+    FROM public.hospitalization
+    
+    WHERE id in (SELECT hospitalization_id FROM public.cases
+      WHERE deleted = FALSE 
+      and caseclassification != 'NO_CASE'
+      and reportdate between '", fromDate, "' and '", toDate, "' 
+      and disease = 'CORONAVIRUS')
+                            ")
+  # Only phospitalizations with a hospitalization_id that also appears in a case 
+  # are queried from the database
+  hospitalizations <- dbGetQuery(sormas_db, queryHospitalizations)
+  
+  # District table SQL query
+  queryDistrict <- paste0("SELECT DISTINCT id AS district_id,
+    name AS district_name
+    
+    FROM public.district
+    
+    WHERE archived = FALSE
+                          ")
+  districts <- dbGetQuery(sormas_db, queryDistrict)
+  
+  # Region table SQL query
+  queryRegions <- paste0("SELECT DISTINCT id AS region_id,
+    name AS region_name
+    
+    FROM public.region 
+    
+    WHERE archived = FALSE
+                         ")
+  regions <- dbGetQuery(sormas_db, queryRegions)
+  
+  # Populationdata table SQL query 
+  queryPopulation <- paste0("SELECT DISTINCT id AS population_id,
```

Comment:
No all sormas instances would have this table, to prevent casexport from breaking, I would suggest that we move the population export to its own export function "populationExport"